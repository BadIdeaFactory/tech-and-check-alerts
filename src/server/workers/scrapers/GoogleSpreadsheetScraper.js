import AbstractScraper from './AbstractScraper'
import { STATEMENT_SCRAPER_NAMES } from './constants'
import {
  getSpreadsheetCsvUrl,
  parseCsvString,
} from '../../utils/google'

class GoogleSpreadsheetScraper extends AbstractScraper {
  constructor(spreadsheetId) {
    super(getSpreadsheetCsvUrl(spreadsheetId))
  }

  getScraperName = () => STATEMENT_SCRAPER_NAMES.GOOGLE_SPREADSHEET

  scrapeHandler = async responseString => parseCsvString(responseString)
}

export default GoogleSpreadsheetScraper
