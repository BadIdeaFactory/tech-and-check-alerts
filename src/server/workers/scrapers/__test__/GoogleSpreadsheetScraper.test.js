import nock from 'nock'
import fs from 'fs'
import path from 'path'

import GoogleSpreadsheetScraper from '../GoogleSpreadsheetScraper'

describe('GoogleSpreadsheetScraper', () => {
  describe('run', () => {
    it('GoogleSpreadsheetScraper should fetch the csv', async () => {
      const spreadsheetMock = fs.readFileSync(path.resolve(__dirname, 'data/spreadsheetMock.csv'), 'utf8')
      nock('https://docs.google.com')
        .get(/.*/)
        .reply(200, spreadsheetMock)

      const spreadsheetScraper = new GoogleSpreadsheetScraper('FakeDocumentId123')
      const result = await spreadsheetScraper.run()
      expect(result).toMatchSnapshot()
    })
  })
})
