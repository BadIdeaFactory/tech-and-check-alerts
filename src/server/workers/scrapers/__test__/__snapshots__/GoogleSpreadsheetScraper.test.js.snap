// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleSpreadsheetScraper run GoogleSpreadsheetScraper should fetch the csv 1`] = `
Array [
  Array [
    "first_name",
    "last_name",
  ],
  Array [
    "Greg",
    "Abbott",
  ],
  Array [
    "Stacey",
    "Abrams",
  ],
  Array [
    "Alma",
    "Adams",
  ],
  Array [
    "Justin",
    "Amash",
  ],
  Array [
    "Jimmy",
    "Anderson",
  ],
  Array [
    "Dave",
    "Aronberg",
  ],
  Array [
    "Ricky",
    "Arriola",
  ],
  Array [
    "Troy",
    "Balderson",
  ],
  Array [
    "Tammy",
    "Baldwin",
  ],
  Array [
    "William",
    "Barber",
  ],
  Array [
    "Bill",
    "Barr",
  ],
  Array [
    "William",
    "Barr",
  ],
  Array [
    "Bill",
    "Barrett",
  ],
  Array [
    "Tom",
    "Barrett",
  ],
  Array [
    "William",
    "Barrett",
  ],
  Array [
    "Xavier",
    "Becerra",
  ],
  Array [
    "Joy",
    "Behar",
  ],
  Array [
    "Michael",
    "Bennet",
  ],
  Array [
    "Paul",
    "Bettencourt",
  ],
  Array [
    "Don",
    "Beyer",
  ],
  Array [
    "Chad",
    "Bicknell",
  ],
  Array [
    "Joe",
    "Biden",
  ],
  Array [
    "Dan",
    "Bishop",
  ],
  Array [
    "Marsha",
    "Blackburn",
  ],
  Array [
    "Ken",
    "Blackwell",
  ],
  Array [
    "César",
    "Blanco",
  ],
  Array [
    "Michael",
    "Bloomberg",
  ],
  Array [
    "Dan",
    "Blue",
  ],
  Array [
    "Roy",
    "Blunt",
  ],
  Array [
    "Jair",
    "Bolsanaro",
  ],
  Array [
    "Pam",
    "Bondi",
  ],
  Array [
    "Cory",
    "Booker",
  ],
  Array [
    "Mike",
    "Bost",
  ],
  Array [
    "Kevin",
    "Brady",
  ],
  Array [
    "Dave",
    "Brat",
  ],
  Array [
    "Phil",
    "Bredesen",
  ],
  Array [
    "Harry",
    "Brown",
  ],
  Array [
    "Jerry",
    "Brown",
  ],
  Array [
    "John",
    "Bucy III",
  ],
  Array [
    "Waldemar",
    "Buda",
  ],
  Array [
    "Ted",
    "Budd",
  ],
  Array [
    "Steve",
    "Bullock",
  ],
  Array [
    "Michael",
    "Burgess",
  ],
  Array [
    "Pete",
    "Buttigieg",
  ],
  Array [
    "Donna",
    "Campbell",
  ],
  Array [
    "Maria",
    "Cantwell",
  ],
  Array [
    "Tucker",
    "Carlson",
  ],
  Array [
    "Mitch",
    "Carmichael",
  ],
  Array [
    "Ben",
    "Carson",
  ],
  Array [
    "Greg",
    "Casar",
  ],
  Array [
    "Cindi",
    "Castilla",
  ],
  Array [
    "Julián",
    "Castro",
  ],
  Array [
    "Liz",
    "Cheney",
  ],
  Array [
    "Gery",
    "Chico",
  ],
  Array [
    "David",
    "Cicilline",
  ],
  Array [
    "Pat",
    "Cipollone",
  ],
  Array [
    "David",
    "Clarke",
  ],
  Array [
    "Bill",
    "Clinton",
  ],
  Array [
    "Hillary",
    "Clinton",
  ],
  Array [
    "Ta-Nehisi",
    "Coates",
  ],
  Array [
    "Tom",
    "Cole",
  ],
  Array [
    "Chris",
    "Collins",
  ],
  Array [
    "Doug",
    "Collins",
  ],
  Array [
    "Susan",
    "Collins",
  ],
  Array [
    "Mike",
    "Conaway",
  ],
  Array [
    "Kellyanne",
    "Conway",
  ],
  Array [
    "Roy",
    "Cooper",
  ],
  Array [
    "Richard",
    "Cordray",
  ],
  Array [
    "Josh",
    "Cornett",
  ],
  Array [
    "John",
    "Cornyn",
  ],
  Array [
    "Tom",
    "Cotton",
  ],
  Array [
    "Ann",
    "Coulter",
  ],
  Array [
    "Kevin",
    "Cramer",
  ],
  Array [
    "Brandon",
    "Creighton",
  ],
  Array [
    "Dan",
    "Crenshaw",
  ],
  Array [
    "Ray",
    "Cross",
  ],
  Array [
    "David",
    "Crowley",
  ],
  Array [
    "Ted",
    "Cruz",
  ],
  Array [
    "Mark",
    "Cuban",
  ],
  Array [
    "Ken",
    "Cuccinelli",
  ],
  Array [
    "Henry",
    "Cuellar",
  ],
  Array [
    "Cal",
    "Cunningham",
  ],
  Array [
    "Joe",
    "Cunningham",
  ],
  Array [
    "Andrew",
    "Cuomo",
  ],
  Array [
    "Chris",
    "Cuomo",
  ],
  Array [
    "Steve",
    "Daines",
  ],
  Array [
    "Alberta",
    "Darling",
  ],
  Array [
    "Paul",
    "Davis",
  ],
  Array [
    "Bill",
    "de Blasio",
  ],
  Array [
    "John",
    "Delaney",
  ],
  Array [
    "Antonio",
    "Delgado",
  ],
  Array [
    "Val",
    "Demings",
  ],
  Array [
    "Ron",
    "DeSantis",
  ],
  Array [
    "Ted",
    "Deutch",
  ],
  Array [
    "Betsy",
    "DeVos",
  ],
  Array [
    "Mike",
    "DeWine",
  ],
  Array [
    "David",
    "DiPietro",
  ],
  Array [
    "Lou",
    "Dobbs",
  ],
  Array [
    "Joe",
    "Donnelly",
  ],
  Array [
    "Michael",
    "Dougherty",
  ],
  Array [
    "Kevin",
    "Downing",
  ],
  Array [
    "Ainsley",
    "Earhadt",
  ],
  Array [
    "Keith",
    "Edwards",
  ],
  Array [
    "Joni",
    "Ernst",
  ],
  Array [
    "Jon",
    "Erpenbach",
  ],
  Array [
    "Veronica",
    "Escobar",
  ],
  Array [
    "Tony",
    "Evers",
  ],
  Array [
    "Patricia ",
    "Fahy",
  ],
  Array [
    "Gary",
    "Farmer",
  ],
  Array [
    "Anthony",
    "Fauci",
  ],
  Array [
    "Russ",
    "Feingold",
  ],
  Array [
    "Dianne ",
    "Feinstein",
  ],
  Array [
    "Joshua",
    "Feuerstein",
  ],
  Array [
    "Abby",
    "Finkenauer",
  ],
  Array [
    "Scott",
    "Fitzgerald",
  ],
  Array [
    "Jeff",
    "Flake",
  ],
  Array [
    "John",
    "Flanagan",
  ],
  Array [
    "Ari",
    "Fleischer",
  ],
  Array [
    "Ryan",
    "Fournier",
  ],
  Array [
    "Al",
    "Franken",
  ],
  Array [
    "Nikki",
    "Fried",
  ],
  Array [
    "Marcia",
    "Fudge",
  ],
  Array [
    "Tulsi",
    "Gabbard",
  ],
  Array [
    "Matt",
    "Gaetz",
  ],
  Array [
    "Mike",
    "Gallagher",
  ],
  Array [
    "Gordon",
    "Gee",
  ],
  Array [
    "Paul",
    "Gigot",
  ],
  Array [
    "Kristen",
    "Gillibrand",
  ],
  Array [
    "Andrew",
    "Gillum",
  ],
  Array [
    "Rudolph",
    "Giuliani",
  ],
  Array [
    "Rudy",
    "Giuliani",
  ],
  Array [
    "Louie ",
    "Gohmert",
  ],
  Array [
    "Whoopi",
    "Goldberg",
  ],
  Array [
    "Lindsey",
    "Graham",
  ],
  Array [
    "Mark",
    "Green",
  ],
  Array [
    "Stan",
    "Greenberg",
  ],
  Array [
    "Joel",
    "Griffith",
  ],
  Array [
    "Stephanie",
    "Grisham",
  ],
  Array [
    "Glenn",
    "Grothman",
  ],
  Array [
    "Joe",
    "Gruters",
  ],
  Array [
    "Warren",
    "Gunnels",
  ],
  Array [
    "Brad",
    "Halbrook",
  ],
  Array [
    "Nikki",
    "Haley",
  ],
  Array [
    "Chelsea",
    "Handler",
  ],
  Array [
    "Sean",
    "Hannity",
  ],
  Array [
    "Kamala",
    "Harris",
  ],
  Array [
    "Mark",
    "Harris",
  ],
  Array [
    "James",
    "Hauser",
  ],
  Array [
    "Josh",
    "Hawley",
  ],
  Array [
    "Carl",
    "Heastie",
  ],
  Array [
    "M.J.",
    "Hegar",
  ],
  Array [
    "Heidi",
    "Heitkamp",
  ],
  Array [
    "Dean",
    "Heller",
  ],
  Array [
    "Mac",
    "Heller",
  ],
  Array [
    "John",
    "Hickenlooper",
  ],
  Array [
    "Fiona",
    "Hill",
  ],
  Array [
    "Katie",
    "Hill",
  ],
  Array [
    "Steve",
    "Hilton",
  ],
  Array [
    "Jim",
    "Himes",
  ],
  Array [
    "Mazie",
    "Hirono",
  ],
  Array [
    "Renee",
    "Hoagenson",
  ],
  Array [
    "Robert",
    "Hockett",
  ],
  Array [
    "Donna",
    "Howard",
  ],
  Array [
    "Steny",
    "Hoyer",
  ],
  Array [
    "Sarah",
    "Huckabee Sanders",
  ],
  Array [
    "Richard ",
    "Hudson",
  ],
  Array [
    "Jim",
    "Humble",
  ],
  Array [
    "Rob",
    "Hutton",
  ],
  Array [
    "Laura",
    "Ingraham",
  ],
  Array [
    "Jay",
    "Inslee",
  ],
  Array [
    "Darren",
    "Jackson",
  ],
  Array [
    "Jeff",
    "Jackson",
  ],
  Array [
    "Letitia",
    "James",
  ],
  Array [
    "Adam",
    "Jarchow",
  ],
  Array [
    "Hakeem",
    "Jeffries",
  ],
  Array [
    "Evan",
    "Jenkins",
  ],
  Array [
    "Mark",
    "Jewell",
  ],
  Array [
    "Mike",
    "Johnson",
  ],
  Array [
    "Ron",
    "Johnson",
  ],
  Array [
    "Alex",
    "Jones",
  ],
  Array [
    "Kaya",
    "Jones",
  ],
  Array [
    "Jim",
    "Jordan",
  ],
  Array [
    "Brian",
    "Joseph",
  ],
  Array [
    "Jim",
    "Justice",
  ],
  Array [
    "Tim",
    "Kaine",
  ],
  Array [
    "Chris",
    "Kapenga",
  ],
  Array [
    "Jill",
    "Karofsky",
  ],
  Array [
    "Josh",
    "Kaul",
  ],
  Array [
    "Brett",
    "Kavanaugh",
  ],
  Array [
    "Michael",
    "Kearns",
  ],
  Array [
    "Brendan",
    "Kelly",
  ],
  Array [
    "Brian",
    "Kemp",
  ],
  Array [
    "Joe",
    "Kennedy",
  ],
  Array [
    "John",
    "Kennedy",
  ],
  Array [
    "Ro",
    "Khanna",
  ],
  Array [
    "Jimmy",
    "Kimmel",
  ],
  Array [
    "Ron",
    "Kind",
  ],
  Array [
    "Steve",
    "King",
  ],
  Array [
    "Charlie",
    "Kirk",
  ],
  Array [
    "Amy",
    "Klobuchar",
  ],
  Array [
    "Brian",
    "Kolb",
  ],
  Array [
    "Brian",
    "Kolfage",
  ],
  Array [
    "Dale",
    "Kooyenga",
  ],
  Array [
    "Larry",
    "Kudlow",
  ],
  Array [
    "Jared",
    "Kushner",
  ],
  Array [
    "Ricardo",
    "Lara",
  ],
  Array [
    "Troy",
    "LaRaviere",
  ],
  Array [
    "Chris",
    "Larson",
  ],
  Array [
    "Patrick",
    "Leahy",
  ],
  Array [
    "Mark",
    "Levin",
  ],
  Array [
    "Corey",
    "Lewandowski",
  ],
  Array [
    "David R.",
    "Lewis",
  ],
  Array [
    "Lori",
    "Lightfoot",
  ],
  Array [
    "Rush",
    "Limbaugh",
  ],
  Array [
    "Michael",
    "Long",
  ],
  Array [
    "Barry",
    "Loudermilk",
  ],
  Array [
    "Nita",
    "Lowey",
  ],
  Array [
    "Rachel",
    "Maddow",
  ],
  Array [
    "Gae",
    "Magnafici",
  ],
  Array [
    "Michelle",
    "Malkin",
  ],
  Array [
    "Joe",
    "Manchin",
  ],
  Array [
    "Paul",
    "Maner",
  ],
  Array [
    "Kathy",
    "Manning",
  ],
  Array [
    "Natasha",
    "Marcus",
  ],
  Array [
    "Roland",
    "Martin",
  ],
  Array [
    "Kevin",
    "McAleenan",
  ],
  Array [
    "Terry",
    "McAuliffe",
  ],
  Array [
    "Kevin",
    "McCarthy",
  ],
  Array [
    "Claire",
    "McCaskill",
  ],
  Array [
    "Tom",
    "McClintock",
  ],
  Array [
    "Mitch",
    "McConnell",
  ],
  Array [
    "Dan",
    "McCready",
  ],
  Array [
    "Kayleigh",
    "McEnany",
  ],
  Array [
    "Brett",
    "McGurk",
  ],
  Array [
    "David",
    "McKinley",
  ],
  Array [
    "Nate",
    "McMurray",
  ],
  Array [
    "Nathan",
    "McMurray",
  ],
  Array [
    "Martha",
    "McSally",
  ],
  Array [
    "Mark",
    "Meadows",
  ],
  Array [
    "Sara",
    "Meaney",
  ],
  Array [
    "David",
    "Menchetti",
  ],
  Array [
    "Susana",
    "Mendoza",
  ],
  Array [
    "Jeff",
    "Merkley",
  ],
  Array [
    "Wayne",
    "Messam",
  ],
  Array [
    "Alyssa",
    "Milano",
  ],
  Array [
    "Carol",
    "Miller",
  ],
  Array [
    "Quentin",
    "Miller",
  ],
  Array [
    "Sid",
    "Miller",
  ],
  Array [
    "Stephen",
    "Miller",
  ],
  Array [
    "Andrea",
    "Mitchell",
  ],
  Array [
    "Steven",
    "Mnuchin",
  ],
  Array [
    "Ashley",
    "Moody",
  ],
  Array [
    "Gwen",
    "Moore",
  ],
  Array [
    "Michael",
    "Moore",
  ],
  Array [
    "Tim",
    "Moore",
  ],
  Array [
    "Shelley",
    "Moore Capito",
  ],
  Array [
    "Patrick",
    "Morrisey",
  ],
  Array [
    "Brandon",
    "Morse",
  ],
  Array [
    "Mike",
    "Most",
  ],
  Array [
    "Seth",
    "Moulton",
  ],
  Array [
    "Mick",
    "Mulvaney",
  ],
  Array [
    "Chris",
    "Murphy",
  ],
  Array [
    "Greg",
    "Murphy",
  ],
  Array [
    "Patrick",
    "Murphy",
  ],
  Array [
    "Jerrold",
    "Nadler",
  ],
  Array [
    "Jerry",
    "Nadler",
  ],
  Array [
    "Andrew",
    "Napolitano",
  ],
  Array [
    "Peter",
    "Navarro",
  ],
  Array [
    "Scott",
    "Neitzel",
  ],
  Array [
    "Bill",
    "Nelson",
  ],
  Array [
    "Neon",
    "Nettle",
  ],
  Array [
    "Gavin",
    "Newsom",
  ],
  Array [
    "Wiley",
    "Nickle",
  ],
  Array [
    "Kirstjen",
    "Nielsen",
  ],
  Array [
    "Ralph",
    "Northam",
  ],
  Array [
    "Devin",
    "Nunes",
  ],
  Array [
    "John",
    "Nygren",
  ],
  Array [
    "Robert",
    "O'Brien",
  ],
  Array [
    "Tim",
    "O'Brien",
  ],
  Array [
    "Danny",
    "O'Connor",
  ],
  Array [
    "Beto",
    "O'Rourke",
  ],
  Array [
    "Barack",
    "Obama",
  ],
  Array [
    "Alexandria",
    "Ocasio-Cortez",
  ],
  Array [
    "Richard",
    "Ojeda",
  ],
  Array [
    "Ilhan",
    "Omar",
  ],
  Array [
    "Jim",
    "Ott",
  ],
  Array [
    "Candace",
    "Owens",
  ],
  Array [
    "Rickey",
    "Padgett",
  ],
  Array [
    "Weston",
    "Pagano",
  ],
  Array [
    "Lizzie",
    "Pannill Fletcher",
  ],
  Array [
    "Arlen",
    "Parsa",
  ],
  Array [
    "Brad",
    "Parscale",
  ],
  Array [
    "Dan",
    "Patrick",
  ],
  Array [
    "Rand",
    "Paul",
  ],
  Array [
    "Ron",
    "Paul",
  ],
  Array [
    "Ken",
    "Paxton",
  ],
  Array [
    "Charles",
    "Payne",
  ],
  Array [
    "Nancy",
    "Pelosi",
  ],
  Array [
    "Mike",
    "Pence",
  ],
  Array [
    "Crystal",
    "Peoples-Stokes",
  ],
  Array [
    "David",
    "Perdue",
  ],
  Array [
    "Mark",
    "Perry",
  ],
  Array [
    "Patrick",
    "Philbin",
  ],
  Array [
    "Jeanine",
    "Pirro",
  ],
  Array [
    "Valerie",
    "Plame",
  ],
  Array [
    "Danielle",
    "Pletka",
  ],
  Array [
    "Mark",
    "Pocan",
  ],
  Array [
    "Mike",
    "Pompeo",
  ],
  Array [
    "Katie",
    "Porter",
  ],
  Array [
    "David",
    "Price",
  ],
  Array [
    "J.B.",
    "Pritzker",
  ],
  Array [
    "Scott",
    "Pruitt",
  ],
  Array [
    "Aftab",
    "Pureval",
  ],
  Array [
    "Michael",
    "Purpura",
  ],
  Array [
    "Mike",
    "Purpura",
  ],
  Array [
    "Vladimir",
    "Putin",
  ],
  Array [
    "Rick",
    "Range",
  ],
  Array [
    "Jane",
    "Raskin",
  ],
  Array [
    "John",
    "Ratcliffe",
  ],
  Array [
    "Bruce",
    "Rauner",
  ],
  Array [
    "Tom",
    "Reed",
  ],
  Array [
    "Terrie",
    "Rizzo",
  ],
  Array [
    "Cecil",
    "Roberts",
  ],
  Array [
    "Mike",
    "Romano",
  ],
  Array [
    "Jacky",
    "Rosen",
  ],
  Array [
    "Roger",
    "Roth",
  ],
  Array [
    "Marco",
    "Rubio",
  ],
  Array [
    "Stephanie",
    "Ruhle",
  ],
  Array [
    "Tim",
    "Ryan",
  ],
  Array [
    "Chris",
    "Salcedo",
  ],
  Array [
    "Jack",
    "Salzwedel",
  ],
  Array [
    "Jeremiah",
    "Samples",
  ],
  Array [
    "Robert",
    "Sanborn",
  ],
  Array [
    "Bernie",
    "Sanders",
  ],
  Array [
    "Sarah",
    "Huckabee Sanders",
  ],
  Array [
    "Bennett",
    "Sandlin",
  ],
  Array [
    "John",
    "Sandweg",
  ],
  Array [
    "Joe",
    "Sanfelippo",
  ],
  Array [
    "Andrew",
    "Saul",
  ],
  Array [
    "Steve",
    "Scalise",
  ],
  Array [
    "Anthony",
    "Scaramucci",
  ],
  Array [
    "Adam",
    "Schiff",
  ],
  Array [
    "Curt",
    "Schilling",
  ],
  Array [
    "Brad",
    "Schimel",
  ],
  Array [
    "Howard",
    "Schultz",
  ],
  Array [
    "Charles",
    "Schumer",
  ],
  Array [
    "Chuck",
    "Schumer",
  ],
  Array [
    "Arnold",
    "Schwarzenegger",
  ],
  Array [
    "Bobby",
    "Scott",
  ],
  Array [
    "Rick",
    "Scott",
  ],
  Array [
    "Louis",
    "Seidman",
  ],
  Array [
    "Jay",
    "Sekulow",
  ],
  Array [
    "Jim",
    "Sensenbrenner",
  ],
  Array [
    "Jeff",
    "Sessions",
  ],
  Array [
    "Donna",
    "Shalala",
  ],
  Array [
    "Mike",
    "Shields",
  ],
  Array [
    "Samuel",
    "Sinyangwe",
  ],
  Array [
    "Adam",
    "Smith",
  ],
  Array [
    "Noah",
    "Smith",
  ],
  Array [
    "Gordon",
    "Sondland",
  ],
  Array [
    "Bob",
    "Spindell",
  ],
  Array [
    "Todd",
    "Starnes",
  ],
  Array [
    "Ken",
    "Starr",
  ],
  Array [
    "Elise",
    "Stefanick",
  ],
  Array [
    "Bryan",
    "Steil",
  ],
  Array [
    "Darrell",
    "Steinberg",
  ],
  Array [
    "Greg",
    "Steube",
  ],
  Array [
    "Tom",
    "Steyer",
  ],
  Array [
    "Roger",
    "Stone",
  ],
  Array [
    "Levar",
    "Stoney",
  ],
  Array [
    "Alexander",
    "Strenger",
  ],
  Array [
    "Eric",
    "Swalwell",
  ],
  Array [
    "Rob",
    "Swearingen",
  ],
  Array [
    "Taylor",
    "Swift",
  ],
  Array [
    "Jake",
    "Tapper",
  ],
  Array [
    "Chris",
    "Taylor",
  ],
  Array [
    "Lena",
    "Taylor",
  ],
  Array [
    "William",
    "Taylor",
  ],
  Array [
    "Jeremy",
    "Thiesfeldt",
  ],
  Array [
    "Woody",
    "Thrasher",
  ],
  Array [
    "Thom",
    "Tillis",
  ],
  Array [
    "Jerry",
    "Tillman",
  ],
  Array [
    "Chuck",
    "Todd",
  ],
  Array [
    "John",
    "Torbett",
  ],
  Array [
    "Donald",
    "Trump",
  ],
  Array [
    "Eric",
    "Trump",
  ],
  Array [
    "Ivanka",
    "Trump",
  ],
  Array [
    "Donald",
    "Trump Jr.",
  ],
  Array [
    "Garland",
    "Tucker",
  ],
  Array [
    "Ron",
    "Tusler",
  ],
  Array [
    "Terry",
    "Van Duyn",
  ],
  Array [
    "Alexander",
    "Vindman",
  ],
  Array [
    "Robin",
    "Vos",
  ],
  Array [
    "Leah",
    "Vukmir",
  ],
  Array [
    "Ann",
    "Wagner",
  ],
  Array [
    "Mark",
    "Walker",
  ],
  Array [
    "Scott",
    "Walker",
  ],
  Array [
    "Nicole",
    "Wallace",
  ],
  Array [
    "Mimi",
    "Walters",
  ],
  Array [
    "Elizabeth",
    "Warren",
  ],
  Array [
    "Leana",
    "Wen",
  ],
  Array [
    "Brad",
    "Wenstrup",
  ],
  Array [
    "Andrew",
    "Wheeler",
  ],
  Array [
    "Frank",
    "White",
  ],
  Array [
    "Sheldon",
    "Whitehouse",
  ],
  Array [
    "Terrence",
    "Williams",
  ],
  Array [
    "Marianne",
    "Williamson",
  ],
  Array [
    "Frederica",
    "Wilson",
  ],
  Array [
    "Jacob",
    "Wohl",
  ],
  Array [
    "Dallas",
    "Woodhouse",
  ],
  Array [
    "Chuck",
    "Woolery",
  ],
  Array [
    "JD",
    "Wooten",
  ],
  Array [
    "Gene",
    "Wu",
  ],
  Array [
    "Andrew",
    "Yang",
  ],
  Array [
    "Ted",
    "Yoho",
  ],
]
`;
